psse/psse-model/src/main/java/com/powsybl/psse/model/io/LegacyTextReader.java

```diff
@@ -123,14 +127,38 @@ public String readRecordLine() throws IOException {
         return newLine.toString().trim();
     }
 
-    // all the lines beginning with "@!" are record lines defining the attribute fields
-    private static boolean isRecordLineDefiningTheAttributeFields(String line) {
-        return line.length() >= 2 && line.substring(0, 2).equals("@!");
+    private static String removeComment(String line) {
+        return line.replaceAll("('[^']*')|([^/']+)|(/.*)", "$1$2");
     }
```

Comment:
maybe some changes to concentrate all patterns related to the quote character in a single place

define in the FileFormat class
`public static final String REMOVE_COMMENT_REGEX = "('[^']*')|([^/']+)|(/.*)";`
and use it here
`return line.replaceAll(FileFormat.REMOVE_COMMENT_REGEX, "$1$2");
`